spec/features/admin_claim_check_spec.rb

```diff
@@ -130,6 +130,8 @@
         let!(:claim) { create(:claim, :submitted, personal_details.merge(bank_sort_code: "582939", bank_account_number: "74727752")) }
 
         scenario "User is informed that the claim cannot be approved" do
+          pending("This needs a new step to be added")
```

Comment:
I'd lean towards squashing these two commits into a single commit which adds this new claim checking task. Reduces churn of adding and removing stuff in the specs, and makes sure that we don't have any commits where the claim checking flow is broken.

(and then the CHANGELOG commit can be squashed into here, since the whole feature is implemented in a single commit)

Reply:
Yeah, fair enough. It worked for me while I was working on it, but no reason why is can't be squashed down